### DISCUSSION TITLE
GeoData not being collected

### DISCUSSION THREAD

**Discussion by ATR-oCiTy** [plausible/analytics#940] [Self-Hosted Support]
![image](https://user-images.githubusercontent.com/42792920/115120373-c0b17e80-9fca-11eb-8195-ec08fc450c5a.png)

I can't see any location data in my analytics dashboard.

![image](https://user-images.githubusercontent.com/42792920/115120398-e179d400-9fca-11eb-8ce9-b5050d898011.png)

Data has been recording for the last two days but no geographic data is visible.
I could use some help.

**Comment by RoelSG** [comment]
You should check the release notes first:
https://github.com/plausible/analytics/releases/tag/v1.3.0

The first container isn't used anymore. I didn't have any problems with disappearing geographic data though.

**Reply by ATR-oCiTy** [reply]
Is there any way to check if  Geographic data or the IP is actually being recorded? I updated to 1.3 but I'm still not getting GeoData. Could I  get some help  troubleshooting this?

**Comment by ATR-oCiTy** [comment]
Where is the geodata stored. Is there any way I can check where it's stored in the DB to narrow down the problem?

**Reply by RoelSG** [reply]
Found mine here:

```
~ $ find . -type f -name "*.mmdb"
./lib/plausible-0.0.1/priv/geodb/dbip-country.mmdb
```

This is inside the plausible container

**Reply by ATR-oCiTy** [reply]
```./lib/plausible-0.0.1/priv/geodb/dbip-country.mmdb```

I got this as the output. I still cant seem to find the issue :(.

**Comment by kerns** [comment]
I don't know how long you've been waiting but it takes at least 24 hours I think to show any logged geo data.

**Reply by ATR-oCiTy** [reply]
It has been running for over 3 days now, still, no logs, is there any way I can check if this is being recorded internally?

**Comment by g105b** [comment]
Hey, are you running this using Docker on your server? If so, I think I have the solution for you.

Docker runs unencrypted, on port 8000 by default. It's likely that you are using a reverse proxy to serve Plausible using HTTPS. If this is the case, this is where your geo-ip data is being dropped.

To fix, add the `X-Forwarded-For` header, and set the value to the client's IP. This is all you need to do.

In nginx, I did this:

`proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;`

Hope this helps.

**Comment by AmreshSinha** [comment]
I am facing the same issue. 

I am running plausible behind Traefik and Traefik seems to add `X-Forwaded-For` header by default.
https://doc.traefik.io/traefik/getting-started/faq/#what-are-the-forwarded-headers-when-proxying-http-requests

I have my plausible instance running for 2 days now. But still, I haven't received any geo-data on the dashboard :/
